app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchmessage } from '../redux/message/api';
```

Comment:
- [ ] `fetchmessage` Path is wrong that's why it is giving an error. Please fix it.
![image](https://user-images.githubusercontent.com/38635158/159363250-8aa43153-a6c5-4366-938f-622ffe1ae2ac.png)